README.md

```diff
@@ -26,9 +26,16 @@ This library is more suited to ETL than interactive queries, since large amounts
 
 You may use this library in your applications with the following dependency information:
 
+### Scala 2.10
```

Comment:
Super-minor nit: mind changing this to be bold text (with `**`) instead of a subheading so that it doesn't mess up my table-of-contents generation?